requirements.txt

```
@@ -3,9 +3,8 @@ bioblend>=1.0.0
 click!=8.0.2
 cwltool>=1.0.20191225192155
 ephemeris>=0.10.3
-galaxy-containers<22.5
-galaxy-tool-util>=21.1.1,<22.5
-galaxy-util>=20.5.0,<22.5
+galaxy-tool-util>=21.1.1,<23.0
```

```suggestion
galaxy-tool-util>=22.5.dev2,<23.0
```
I guess that's needed for the moved galaxy.tool_util.containers